test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp.csproj

```diff
@@ -23,6 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\samples\Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Models\Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Models.csproj" />

```

Comment:
In the integration test app, I'd like to be isolated as much as possible. Would you please remove this reference?